packages/core/src/awsService/accessanalyzer/vue/iamPolicyChecks.vue

```diff
@@ -447,15 +447,15 @@ export default defineComponent({
             this.initialData.checkAccessNotGrantedFilePath = event.target.value
             client
                 .readCustomChecksJsonFile(this.initialData.checkAccessNotGrantedFilePath)
-                .then(response => {
+                .then((response) => {
                     this.initialData.checkAccessNotGrantedActionsTextArea = response.actions
                         ? response.actions.toString()
                         : ''
                     this.initialData.checkAccessNotGrantedResourcesTextArea = response.resources
                         ? response.resources.toString()
                         : ''
                 })
-                .catch(err => console.log(err))
+                .catch((err) => console.log(err))
```

Comment:
changes in this file seem to be made by linter

Reply:
^ Changes pertaining to A2 looks fine, reviewed and deferring approval to Toolkits

Reply:
This is a weird bug and we're not sure why it happens. But this is fine for now.